.tool-versions

```diff
@@ -1,2 +1,2 @@
-erlang 26.0
-elixir 1.14.4
+erlang 25.3.2
```

Comment:
ElixirLS has problems starting with erlang 26.0 - reverting to an earlier version just because of that :) Wasmex itself works fine under 26.0